<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="action_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;控制总界面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>游戏控制界面</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="game_control_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>控制区</string>
     </property>
     <widget class="QLabel" name="paddle_pos_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>障碍物下落速度:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="obst_fall_speed_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_reset">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="timer_display">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_14">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>计时器(秒):</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manual_pause">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>障碍物出现间隔:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="obst_freq_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>光流单屏</string>
     </property>
     <widget class="QSpinBox" name="left_dir_speed_spinbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>左速度</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="right_dir_speed_spinbox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_29">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>右速度</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_light">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_light">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QPushButton" name="single_left_shift">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>左转换</string>
      </property>
     </widget>
     <widget class="QPushButton" name="single_right_shift">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>右转换</string>
      </property>
     </widget>
     <widget class="QLabel" name="RL_left_display">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RL左速度</string>
      </property>
     </widget>
     <widget class="QLabel" name="RL_right_display">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RL右速度</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>数据记录</string>
     </property>
     <widget class="QPushButton" name="stop_recording">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>停止记录</string>
      </property>
     </widget>
     <widget class="QLabel" name="recording_status">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>未记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_recording">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始记录</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="data_record_interval_spinbox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="recording_status_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数据记录间隔：</string>
      </property>
     </widget>
     <widget class="QLabel" name="actual_record_interval_display">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>实际值</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>x位置参数：</string>
     </property>
     <widget class="QLabel" name="paddle_pos_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>最大值: </string>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>数据量:</string>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x坐标放大:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="x_max_spinbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="data_num_spinbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="x_coord_factor_spinbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="x_max_actual">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实际值</string>
      </property>
     </widget>
     <widget class="QLabel" name="paddle_pos_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>扭矩值: </string>
      </property>
     </widget>
     <widget class="QLabel" name="torque_data_avg">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="data_num_display">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实际值</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_coord_factor_display">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实际值</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>300</y>
       <width>191</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>AI控制</string>
     </property>
     <widget class="QPushButton" name="simple_AI">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>简单AI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RL_train">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>自动调参</string>
      </property>
     </widget>
     <widget class="QLabel" name="AI_mode">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>51</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>AI模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RL_test">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>自动避障</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
